server/ai_mediaserver.go

```diff
@@ -482,6 +482,14 @@ func (ls *LivepeerServer) StartLiveVideo() http.Handler {
 			ls.cleanupLive(streamName)
 		}()
 
+		errorCallback := func(err error) {
+			clog.Errorf(ctx, "Failed to process live video: %s", err)
```

Comment:
1. What if error is `nil`, do we still want to log an error?
2. So, no matter if error is nil or not, we kick of the connection? if that's the case, then could you add a comment explaining it? It sounds counter-intutivie.

Reply:
You're right, it can be nil in some cases, I'll improve it

Reply:
I've changed things around so that this should never be called with a nil error but added a nil check anyway. We kick the input connection to allow the client to reconnect and restart the pipeline